openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -34,6 +34,9 @@ pipeline {
     MAINTENANCE_DIRECTORY = "maintenance"
     DEVOPS_DIRECTORY = "openshift"
 
+    // SonarQube configuration
+    SONARQUBE_URL_INT = "http://sonarqube:9000"
```

Comment:
We don't need the external URL here because our sonarqube instance runs on the same openshift namespace/project `<...>-tools`